rollup.config.js

```diff
@@ -0,0 +1,14 @@
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+
+export default {
```

Comment:
What is rollup? Is this like webpack?

Reply:
Similar, yeah. Sort of a "webpack light" from my understanding.